src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -809,7 +809,9 @@ public static void BeginInvokeDotNetFailed(ILogger logger, string callId, string
             }
         }
 
+#pragma warning disable SYSLIB1025
```

Comment:
What is the warning from?

Reply:
Ah, it's from LocationChangeFailed. Should change to LocationChangeFailedInCircuit